src/scene.py

```diff
@@ -0,0 +1,65 @@
+import random
+from src.common.point import Point
+from src.objects.agent import Agent
+from src.objects.floor import Floor
+from src.objects.portal import Portal
+from .objects.object import Object
+
+
+class Scene():
+    '''
+    Object representing one level in the game.
+    It is responsible for storing all instances of `Object` class.
+    '''
+
+    def __init__(self, runtime) -> None:
+        self.objects_count = 0
```

Comment:
Let's create class objects collection aggregating all of `objects` related stuff
count, dict and map

Reply:
Sure, but not now :D [LNX-44](https://gut-lynx.atlassian.net/browse/LNX-44)